syz-cluster/pkg/blob/gcs.go

```diff
@@ -29,54 +29,45 @@ func NewGCSClient(ctx context.Context, bucket string) (Storage, error) {
 	}, nil
 }
 
-func (gcs *gcsDriver) Store(source io.Reader) (string, error) {
-	object := uuid.NewString()
-	err := gcs.writeObject(object, source)
+func (gcs *gcsDriver) NewURI() (string, error) {
+	key, err := uuid.NewRandom()
```

Comment:
uuid.NewString looks better to me. Why uuid.NewRandom?

Reply:
There is at least https://github.com/google/uuid/issues/97 ...

Reply:
> Why uuid.NewRandom

To handle a possible error instead of panicking